app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str):
+    names = command.split()
+    original_name = names[1]
+    file_copy = names[2]
+    if original_name == file_copy:
```

Comment:
use negative result from this statement, and also, you should check the input command